eval.go

```diff
@@ -396,6 +406,75 @@ nextFileMatch:
 	return &res, nil
 }
 
+func chunkMatchesToLineMatches(cms []ChunkMatch, contextLines int) []LineMatch {
+	lms := make([]LineMatch, 0, len(cms))
+	for _, cm := range cms {
+		lines := bytes.Split(cm.Content, []byte("\n"))
```

Comment:
Not necessarily. A single range can cross multiple lines, in which case `cm.Content` will also contain those lines. However, we only ever re-join the lines when `numContextLines > 0`. We could avoid some allocations for `Before`/`After` in that case, but since `numContextLines` is currently unused by Sourcegraph, the simplicity here won out